.github/workflows/production_release.yml

```diff
@@ -0,0 +1,80 @@
+name: Release to production
+
+on:
+  push:
+    tags: '*'
```

Comment:
Will this workflow be triggered for every tag on each branch?

If this is the case, we should state it clearly in the *How to contribute* section, or at least scope it down using a regex, for instance: `^v?(\d+(?:\.\d+)*)$`

Reply:
I see no threats, just risks on maintainers side in case a tag is pushed and an involuntary deployment is triggered.

Anyway, I am good with it :)